experimental/repositories/demologin_repository.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:firebase_auth/firebase_auth.dart';
+import '../models/demouser.dart';
+import 'package:get/get.dart';
+
+class DemologinRepository {
+  static const String _password = "testtest";
+
+  static Future<DemoUser?> authentication(String email) async {
+    try {
+      UserCredential credential = await FirebaseAuth.instance.signInWithEmailAndPassword(
+        email: email,
+        password: _password
```

Comment:
```suggestion
        password: _password,
```
コンマ入れてないと保存した時に改行されないと思うのでコンマ入れておいてほしいです。